### ISSUE TITLE
Example Regression not working in Google Colab

### ISSUE THREAD

**Issue by franz101** [dreamquark-ai/tabnet#117]
**Describe the bug**
I tried to run on gpu and then on cpu and with different embedding sizes.
Still I get a dimension error.

Here is the link to the notebook:
https://colab.research.google.com/drive/1wDQ28PNxtEJA1XZyN2eVA6iTSd6ctf-E?usp=sharing

Maybe related to #94

**Comment by eduardocarvp** [comment]
Hi @franz101, thank you for the bug report.

I will try to reproduce the bug and come back soon with more info on this.

**Comment by franz101** [comment]
Ok meanwhile I just used the preprocessing and the parameters of the classifier notebook which worked. Seems like there is an issue with preprocessing proabably due to pandas 1.
And also the cat_emb_dim is now set to 1

**Comment by Optimox** [comment]
hey @franz101 could you show the error you to, if it's about embedding size with regression we already fixed the bug, but it has not been release yet, so you can use the code from develop branch or wait for the release that should come this week.

**Comment by franz101** [comment]
<img width="1150" alt="Screenshot 2020-06-02 at 12 16 20" src="https://user-images.githubusercontent.com/18228395/83508931-ec1e9e00-a4ca-11ea-9cc5-0e8d6ec146c5.png">

Sure no problem.

I fixed the issues by commenting them out.
First issue was due indexing problems in my dataset (the fillna with mean part)
Second issue was due to the embedding dimensions.
Third issue was due to a reshaping in the Train test data (.reshape(-1, 1))

Thanks @eduardocarvp @Optimox 
The notebook is now working. Btw is there a way of installing the latest dev version with pip install git://.... ?
